File: Corte1/Python/Ejercicio1/Ejercicio1.py

class Libro:
    def __init__(self, titulo, autor):
        self.titulo = titulo
        self.autor = autor

   
    def mostrar_info(self):
        print(f"Título: {self.titulo}")
        print(f"Autor: {self.autor}")
        print("-" * 30)



libro1 = Libro("Cien años de soledad", "Gabriel García Márquez")
libro2 = Libro("1984", "George Orwell")


libro1.mostrar_info()
libro2.mostrar_info()

File: Corte1/Java/Ejercicio1/Ejercicio1.java
class Libro {
    String titulo;
    String autor;


    public Libro(String titulo, String autor) {
        this.titulo = titulo;
        this.autor = autor;
    }


    public void mostrarInfo() {
        System.out.println("Título: " + titulo);
        System.out.println("Autor: " + autor);
        System.out.println("----------------------");
    }
}

public class Main {
    public static void main(String[] args) {

        Libro libro1 = new Libro("Cien años de soledad", "Gabriel García Márquez");
        Libro libro2 = new Libro("1984", "George Orwell");


        libro1.mostrarInfo();
        libro2.mostrarInfo();
    }
}
